### ISSUE TITLE
vcpkg upgrade  Error: Port definition not found.

### ISSUE THREAD

**Issue by Yang09701194** [microsoft/cpprestsdk#1424]
When I execute `vcpkg upgrade`, I got error `Port definition not found.`.

Generally how to solve it? reinstall boost takes lots of time, should I really remove `recurse`? 

Thank you. Cmd:

```
C:\vcpkg>vcpkg upgrade
Error: while loading boost-disjoint-sets:x86-windows: Port definition not found.
Please run "vcpkg remove boost-disjoint-sets:x86-windows" and re-attempt.

C:\vcpkg>vcpkg remove boost-disjoint-sets:x86-windows
The following packages will be removed:
  * boost:x86-windows
    boost-disjoint-sets:x86-windows
Additional packages (*) need to be removed to complete this operation.
If you are sure you want to remove them, run the command with the --recurse option

```

**Comment by Yang09701194** [comment]
https://github.com/microsoft/vcpkg/issues/11463

**Comment by name2name2** [comment]
fixes #1424

**Comment by Yang09701194** [comment]
seems microsoft has fixed it
